packaging/ondemand-bc_osc_rstudio_server.spec

```diff
@@ -2,6 +2,7 @@
 %global debug_package %{nil}
 %global repo_name bc_osc_rstudio_server
 %global app_name bc_osc_rstudio_server
+%define git_tag_minus_v %(echo %{git_tag} | sed -r 's/^v//')
 
 Name:     ondemand-%{app_name}
 Version:  0.8.1
```

Comment:
So it's written, needs to be `${package_version}`

Reply:
Having issues going one line down, but release needs to have `1` replaced with `%{package_release}`